eng/Versions.props

```diff
@@ -47,7 +47,7 @@
     <MicrosoftCodeAnalysisForRoslynDiagnosticsAnalyzersVersion>3.7.0</MicrosoftCodeAnalysisForRoslynDiagnosticsAnalyzersVersion>
     <MicrosoftCodeAnalysisVersionForTests>4.4.0-2.22416.9</MicrosoftCodeAnalysisVersionForTests>
     <DogfoodAnalyzersVersion>3.3.4-beta1.22418.3</DogfoodAnalyzersVersion>
-    <DogfoodNetAnalyzersVersion>5.0.4-preview1.21126.5</DogfoodNetAnalyzersVersion>
+    <DogfoodNetAnalyzersVersion>7.0.0</DogfoodNetAnalyzersVersion>
```

Comment:
Wow, this did not generate any new build warnings from CA diagnostics?

Reply:
It was quite surprising, but yeah no new warnings :)